src/Tree.hpp

```diff
@@ -104,9 +127,14 @@ namespace RRT
             //  default values
             setStepSize(0.1);
             setMaxIterations(1000);
+            setASCEnabled(false);
             setGoalBias(0);
             setWaypointBias(0);
             setGoalMaxDist(0.1);
+            setASCGrowthRate(1.5);
+            _stateSpace->setMinStepSize(.1);
```

Comment:
We probably shouldn't set parameters on the state space here.  If I did something like this:
~~~{.cpp}
StateSpace myStateSpace;
myStateSpace.setMinStepSize(2);
Tree tree(myStateSpace);
~~~

the parameters I set before would get wiped out.